### ISSUE TITLE
In which File I can include CookieBar

### ISSUE THREAD

**Issue by AlexL777** [quickapps/website#7]
Hello Developers,

I use Google Adsense on my QA website and I need to integrate the CookieBar. In which template or which file can I embed the code with CookieBar(Cookie Consent)?

Somehow I do not find the right templates.

Thanks in advance.

**Comment by ChristopherCastro** [comment]
Are you using a custom build theme, or using built-in front theme? Here some tips on how to build your own theme: http://book.quickappscms.org/designers/themes.html

---

If you want to use and "mod" buil-in themes You could try to use site-wide template feature, you must place your template files at `ROOT/templates/Common`. I think I forgot to document this feature on the QuickBook,  I'll write it ASAP. However you can check code's docblock which gives an idea of this feature: https://github.com/quickapps/cms/blob/2.0/plugins/CMS/src/View/View.php#L239

Basically, if you want to "override" any template you can place a moded copy of that template on the "ROOT/templates/" directory. For instance, if you want to customise [this layout](https://github.com/quickapps/cms/blob/2.0/plugins/FrontendTheme/src/Template/Layout/default.ctp), you can create your own at "ROOT/templates/Front/Layout/default.ctp". So whenever QACMS tries to locate the original it will use own instead:

`/FrontendTheme/src/Template/Layout/default.ctp → ROOT/templates/Front/Layout/default.ctp`

Hope this help,
regards!

**Comment by AlexL777** [comment]
Thank you for your help. So I just realized it and it worked. In addition, I have still embedded under http://quickapps.wpzweinull.ch/blog Shariff Share Buttons. I also did that in default.ctp. That worked great. Now I'm an immense step forward 😸
